concepts.tex

```diff
@@ -127,6 +127,209 @@
 which requires \tcode{C<T>()}.
 \exitexample
 
+\rSec1[concepts.lib.synopsis]{Header \tcode{<experimental/ranges/concepts>} synopsis}
+
+\indexlibrary{\idxhdr{experimental/ranges/concepts}}%
+\begin{codeblock}
+namespace std { namespace experimental { namespace ranges { inline namespace v1 {
+  // \ref{concepts.lib.corelang}, core language concepts:
+  // \ref{concepts.lib.corelang.same}, Same:
+  template <class T, class U>
+  concept bool Same() {
```

Comment:
Should we add another indent on these to match the standard's function template declaration style?